tools/bumper/cnao_repo_commands_test.go

```diff
@@ -55,9 +55,7 @@ var _ = Describe("Testing internal git CNAO Repo", func() {
 
 		DescribeTable("and checking isPrAlreadyOpened function",
 			func(r isPrAlreadyOpenedParams) {
-				defer func(path string) {
-					Expect(os.RemoveAll(path)).To(Succeed())
-				}(gitCnaoRepo.gitRepo.localDir)
+				DeferCleanup(os.RemoveAll, gitCnaoRepo.gitRepo.localDir)
```

Comment:
Aint `DeferCleanup` should be used on `BeforeAll` and such ?
It meant to auto create a corresponding `AfterAll` block for example,
Here it feels that defer is more suitable,
unless the docs etc says otherwise

Reply:
No, that ok. See the question here https://gophers.slack.com/archives/CQQ50BBNW/p173627601851229,  and the answer here https://gophers.slack.com/archives/CQQ50BBNW/p1736281169698839